test/worlds/fuel.ignitionrobotics.org/OpenRobotics/worlds/Test%20world/2/test.world

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" ?>
```

Comment:
Let's remove this file. The test worlds on Fuel have it for backwards compatibility, because these files have been there for a long time. But they're not something `ign-gazebo` should support or care about.

Reply:
60d24050f44caa2e2abe4e52163ad600d3869b72